+ source ../common/net_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Aug  8 19:23:07 2023 - INFO  - Start to run test.
+ rm /etc/resolv.conf
+ systemctl reload NetworkManager
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ grep nameserver /etc/resolv.conf
grep: /etc/resolv.conf: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmcli_cancel_auto.sh line 26'
+ message='oe_test_nmcli_cancel_auto.sh line 26'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmcli_cancel_auto.sh line 26'
Tue Aug  8 19:23:09 2023 - ERROR - oe_test_nmcli_cancel_auto.sh line 26
+ '[' 0 -eq 1 ']'
+ return 0
+ echo '[main]
dns=none'
+ rm /etc/resolv.conf
rm: cannot remove '/etc/resolv.conf': No such file or directory
+ systemctl reload NetworkManager
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ test -f /etc/resolv.conf
+ CHECK_RESULT 1 0 1
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ rm /etc/NetworkManager/conf.d/90-dns-none.conf
+ systemctl reload NetworkManager
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ grep nameserver /etc/resolv.conf
grep: /etc/resolv.conf: No such file or directory
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_nmcli_cancel_auto.sh line 36'
+ message='oe_test_nmcli_cancel_auto.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_nmcli_cancel_auto.sh line 36'
Tue Aug  8 19:23:12 2023 - ERROR - oe_test_nmcli_cancel_auto.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ rm /etc/resolv.conf
rm: cannot remove '/etc/resolv.conf': No such file or directory
+ echo '## test'
+ ln -s /etc/resolv.conf.manually-configured /etc/resolv.conf
+ systemctl reload NetworkManager
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ grep test /etc/resolv.conf
## test
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Aug  8 19:23:15 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Aug  8 19:23:18 2023 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue Aug  8 19:23:20 2023 - INFO  - Start to restore the test environment.
+ rm -rf /etc/resolv.conf.manually-configured
+ rm -rf /etc/resolv.conf
+ systemctl reload NetworkManager
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue Aug  8 19:23:23 2023 - INFO  - End to restore the test environment.
